l10n_it_intrastat/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,113 @@
+**Italiano**
+
+In Configurazione --> Aziende --> Aziende --> <company>
+impostare i parametri delle sezioni presenti nel tab Configurazione:
+
+1. Intrastat
+
+   a) User ID (UA Code): inserire il codice identificativo Intrastat dell’azienda (codice alfanumerico di 4 caratteri, utilizzato come user-id per l’accesso alle applicazioni delle Dogane)
```

Comment:
ID utente (codice UA)

Ho notato che molte altre voci in questo file sono lasciate in inglese.

Reply:
fatto